ubuntu/latest.Dockerfile

```
@@ -7,13 +7,151 @@
+        postgresql-client \
+        default-jre \
+        default-jdk \
+        chromium-bsu \
+        chromium-browser \
+        xvfb \
+        build-essential cmake libsdl2-dev libfreetype6-dev libglew-dev libglm-dev && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)